shenyu-admin/src/main/resources/sql-script/h2/schema.sql

```diff
@@ -354,6 +354,19 @@ INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_v
 INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_value`, `desc`, `sort`, `enabled`) VALUES ('1545811989312315392', 'apidoc', 'API_DOC_GLOBAL_FLAG', 'status', 'true', 'Global switching (on or off) of API documents.', 0, 1);
 INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_value`, `desc`, `sort`, `enabled`) VALUES ('1545812101056962560', 'apidocEnv', 'ENV_LABEL_OFFLINE', 'Offline', 'http://127.0.0.1:9195', 'Offline environment', 0, 1);
 INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_value`, `desc`, `sort`, `enabled`) VALUES ('1545812228228259840', 'apidocEnv', 'ENV_LABEL_ONLINE', 'Online', 'http://127.0.0.1:9196', 'Online environment', 1, 1);
+INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_value`, `desc`, `sort`, `enabled`) VALUES ('1529402613195784271', 'securityProtocol', 'SECURITY_PROTOCOL', 'default', '', '', 0, 1);
```

Comment:
sql maybe add in mysql，pg，and oracle

Reply:
ok, i will perfect it

Reply:
Hi， 

Since 2.5.0 has been released, the new SQL statement should.

1.Add into shenyu/shenyu-admin/src/resource/sql-script.h2/schema.sql.
2.Add into /shenyu/db/init/* .
3.Create or Update  SQL into 2.5.0-upgrade-2.5.1-pg.sql, 2.5.0-upgrade-2.5.1-mysql.sql , 2.5.0-upgrade-2.5.1-oracle.sql file .

Reply:
Hi yu,sql is perfected,pls Review and Comment